js/Create_Phone_Number.js

```diff
@@ -0,0 +1,11 @@
+// https://www.codewars.com/kata/create-phone-number/train/javascript
+
+function createPhoneNumber(numbers){
+    var numbersArray = numbers;
+    console.log('start');
+    if (numbersArray.length === 10){
+      return ('('+numbersArray[0]+numbersArray[1]+numbersArray[2]+')'+' '+numbersArray[3]+numbersArray[4]+numbersArray[5]+'-'+numbersArray[6]+numbersArray[7]+numbersArray[8]+numbersArray[9]);
```

Comment:
While this works fine, it could be improved to be a bit more readable :slightly_smiling_face:

There are the following methods that are available on all arrays:
1. [`Array.prototype.slice`](https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Obiekty/Array/slice)
2. [`Array.prototype.join`](https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Obiekty/Array/join)

Read up on them and tell me how they could be used to first create the parts of 3 or 4 digits and then concatenate them

---

Additionally, is there an ES6 feature that could be used instead of string concatenation (i.e. how to replace `a +  ' ' + b + ' ' + c`)?

Reply:
I will create 3 arrays and than i will use join metod on each one. Then i will merge them.